### PR TITLE
Merge Release Updates

### PR DESCRIPTION
**Pull request by tomswartz07**
## Overview:

Documentation updates for the merge release.

## Added:

- Updated layouts for readability improvements
- New parameters for service creation
- Security and Compliance articles
- Disaster Recovery articles
- Fix several typos

## Testing Method:
```sh
# Ensure Ruby 2.3 and reqs are installed
cd docs-crunchy-data/docs-book
bundle exec bookbinder bind local
cd final_app
cf push -f ../manifest.yml
# Observe and proof deployed app
```

### CODE REVIEW

**Comment by kyle-nevins** [comment]
A note that there is no manifest.yml file in this repo. I used the one at:
https://github.com/CrunchyData/pcf-doc/blob/feature/merge_docs/crunchy-postgresql-for-pcf/manifest.yml
and it worked fine.

**Comment by tomswartz07** [comment]
That is expected- we do not provide the manifest for this, since it's processed upstream at Pivotal.

File: docs-content/platform-inspec.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,235 @@
+---
```
Comment (jessemcfarland) [inline docs-content/platform-inspec.html.md.erb]: I don't think we need the ATO procedures in the GA release docs, thoughts?

Comment (tomswartz07) [reply docs-content/platform-inspec.html.md.erb]: I included them as some ga release users might want to verify the results internally. 

Doing so, however would mean that we publish the ato tarball on the Crunchy Portal- not that big of an issue, but just to be aware.

Comment (jessemcfarland) [reply docs-content/platform-inspec.html.md.erb]: The procedure relies on the `inspec2ckl-release` which is a one-off and not something we officially support. Maybe we can include something about retrieving a compliance report instead?

Comment (tomswartz07) [reply docs-content/platform-inspec.html.md.erb]: I'll omit it for now, we could work up the documentation and officially supported method of retrieving the inspec results.

**Pull request merged**